pkg/webhook/controlplane/ensurer.go

```diff
@@ -338,44 +342,30 @@ func ensureKubeAPIServerCommandLineArgs(c *corev1.Container, makeAuditForwarder
 }
 
 func ensureAuditForwarder(ps *corev1.PodSpec, auditToSplunk bool) error {
+	auditForwarderSidecar := auditForwarderSidecarTemplate
```

Comment:
To be on the safe side, I'd call the DeepCopy function on the template:

```suggestion
	auditForwarderSidecar := auditForwarderSidecarTemplate.DeepCopy()
```